packages/api/src/target-group/target-groups.service.ts

```diff
@@ -106,22 +106,18 @@ export class TargetGroupsService {
         return true;
     }
 
-    async findNonMainTargetGroups({
+    async findTargetGroups({
         scope,
         offset,
         limit,
+        where,
     }: {
         scope?: EmailCampaignScopeInterface;
```

Comment:
The scope is not used anymore.

I would remove it here, but pass it in the "where" prop